autopush/main.py

```diff
@@ -500,8 +501,10 @@ def endpoint_main(sysargs=None, use_files=True):
 
     # Endpoint HTTP router
     site = cyclone.web.Application([
-        (r"/push/(?:(v\d+)\/)?([^\/]+)", EndpointHandler,
+        (r"/push/(?:(?P<api_ver>v\d+)\/)?(?P<token>[^\/]+)", EndpointHandler,
          dict(ap_settings=settings)),
+        (r"/spush/(?:(?P<api_ver>v\d+)\/)?(?P<token>[^\/]+)",
```

Comment:
We will eventually invalidate them, thus adding the version to new user records. And later, we can then force users without a record_version to get a new UAID which will invalidate existing simplepush /push endpoints.